cpp/ql/src/utils/modelgenerator/GenerateFlowModel.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python3
+
+import sys
+import os.path
+import subprocess
+
+# Add Model as Data script directory to sys.path.
+gitroot = subprocess.check_output(["git", "rev-parse", "--show-toplevel"]).decode("utf-8").strip()
+madpath = os.path.join(gitroot, "misc/scripts/models-as-data/")
+sys.path.append(madpath)
+
+import generate_flow_model as model
+
+language = "cpp"
+model.Generator.make(language).run()
```

Comment:
The python script puts the generated models in
```
lib/ext/generated
```
If you intend to use this location for generated models, maybe consider to extend the `qlpack.yml` to also include data extensions from this location.

Reply:
Oh, right. I've fixed this in a06d7fcd3c8f7a5c070bcaf8394828fe2479fcd7